.github/workflows/php-lint-test.yml

```diff
@@ -4,8 +4,8 @@ on:
   pull_request
 
 env:
+  WC_VERSION: 5.6.0  # the min supported version as per L-2 policy
```

Comment:
Nitpick: may only change the version number rather than change its line?

Reply:
Fixed